packages/uikit-workshop/src/scripts/components/pl-search/pl-search.js

```diff
@@ -10,7 +10,7 @@ import Mousetrap from 'mousetrap';
 import VisuallyHidden from '@reach/visually-hidden';
 import Autosuggest from 'react-autosuggest';
 
-import { urlHandler } from '../../utils';
+import { urlHandler, iframeMsgDataExtraction } from '../../utils';
```

Comment:
+1 on moving this logic out out into `utils`. Been itching to do this but hadn't gotten around to it!